src/main/java/com/kainos/ea/exception/UserAlreadyExistsException.java

```diff
@@ -0,0 +1,8 @@
+package com.kainos.ea.exception;
```

Comment:
I don't see this being thrown anywhere? But it should be :)

Reply:
Now thrown in UserDao